src/views/components/calendar/CalendarHeader/CalendarHeader.tsx

```diff
@@ -98,6 +98,18 @@ export default function CalendarHeader({ sidebarOpen, onSidebarToggle }: Calenda
                                         Save as .cal
                                     </Button>
                                 </MenuItem>
+                                <MenuItem>
+                                    <Button
+                                        className='w-full flex justify-start'
+                                        onClick={() => handleExportJson(getActiveSchedule().id)}
```

Comment:
We already have the line `const [activeSchedule] = useSchedules();`, so I think we can reuse the `activeSchedule` object there instead of getting it again
```suggestion
                                        onClick={() => handleExportJson(activeSchedule.id)}
```
if you follow this suggestion remember to also remove `getActiveSchedule` from the imports